libs/blocks/quiz/utils.js

```diff
@@ -1,10 +1,12 @@
 import { getMetadata } from '../section-metadata/section-metadata.js';
+import { getConfig } from '../../utils/utils.js';
 
 const QUESTIONS_EP_NAME = 'questions.json';
 const STRINGS_EP_NAME = 'strings.json';
 const RESULTS_EP_NAME = 'results.json';
 
 let getConfigPath; let getQuizKey; let getAnalyticsType; let getAnalyticsQuiz; let metaData;
```

Comment:
Just a minor code smell: please do not use `get` prefixed variable names, it's confusing to see getter function naming for regular variables.

Reply:
> Just a minor code smell: please do not use `get` prefixed variable names, it's confusing to see getter function naming for regular variables.

@sabyamon , are you planning to address the requested change? I think it would make the code much more readable.

Reply:
Hi @honstar - Thanks much for the detailed review. It absolutely makes sense and there are more occurrences of similar naming issues in the question flow. We discussed this in our today's planning session and created a follow-up ticket to address this. Hope that works.

https://jira.corp.adobe.com/browse/MWPW-132229